multivariate_grouped_data_plot.Rmd

```diff
@@ -0,0 +1,825 @@
+# How to plot grouped data for multivariate (in Chinese)
+
+Yihui Hu, Liuxin Chen
+
+原始网页 Original website：http://www.sthda.com/english/articles/32-r-graphics-essentials/132-plot-grouped-data-box-plot-bar-plot-and-more/#grouped-continuous-variables
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+
+## 1 数据准备
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings.